engine/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateMigrationService.java

```diff
@@ -546,12 +479,13 @@ private String loadResourceTemplate(String name, String extension) {
      */
     private void createInstantEmailTemplate(List<String> warnings, String bundleName, String appName, List<String> eventTypeNames,
             String subjectTemplateName, String subjectTemplateExtension, String subjectTemplateDescription,
-            String bodyTemplateName, String bodyTemplateExtension, String bodyTemplateDescription, boolean useTemplateV2) {
+            String bodyTemplateName, String bodyTemplateExtension, String bodyTemplateDescription) {
 
-        if (useTemplateV2) {
+        if (!featureFlipper.isUseSecuredEmailTemplates()) {
```

Comment:
Today only Secured env are using templates without V2 extension.
They are V2 indeed but because it was created after V2 migration, we didn't add this extention on them.

V2 extension will have to be removed on RHCLOUD-26726. I didn't did it on this PR because it has already lot of file modified :)

Reply:
thanks!